local-cli/templates/HelloWorld/android/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,3 @@
 <resources>
-    <string name="app_name">HelloWorld</string>
+    <string name="app_name">Hello App Name</string>
```

Comment:
Maybe better use "Hello World"? Is this what's displayed on the home screen? I wouldn't get what "Hello App Name" means.